python-checks/src/test/resources/checks/tooManyLinesInFunctionCustom.py

```diff
@@ -0,0 +1,22 @@
+# Noncompliant@+1 {{This function "function_6" has 6 lines of code, which is greater than the 5 authorized. Split it into smaller functions.}}
```

Comment:
I feel that our unit tests are a bit weak. I would like to see more cases covered where each case differs from another with exactly one difference. We may discuss about it.